inlang/source-code/website/vite.config.ts

```diff
@@ -9,6 +9,9 @@ import { nodePolyfills } from "vite-plugin-node-polyfills"
 const isProduction = process.env.NODE_ENV === "production"
 
 export default defineConfig({
+	server: {
+		port: 4002,
+	},
 	envPrefix: "PUBLIC_",
 	plugins: [
 		nodePolyfills({
```

Comment:
@NiklasBuchfink The node polyfills shouldn't be required for the website. If they are, we screwed up :D